ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BenchmarkSubCommand.java

```diff
@@ -107,13 +118,17 @@ public BenchmarkSubCommand(final PrintStream output) {
   @Override
   public void run() {
     LogConfigurator.setLevel("", "DEBUG");
-    System.out.println(BesuVersionUtils.version());
+    output.println(BesuVersionUtils.version());
     AbstractPrecompiledContract.setPrecompileCaching(enablePrecompileCache);
     AbstractBLS12PrecompiledContract.setPrecompileCaching(enablePrecompileCache);
     var benchmarksToRun = benchmarks.isEmpty() ? EnumSet.allOf(Benchmark.class) : benchmarks;
     for (var benchmark : benchmarksToRun) {
-      System.out.println("Benchmarks for " + benchmark);
-      benchmark.benchmarkExecutor.runBenchmark(output, nativeCode, parentCommand.getFork());
+      output.println("Benchmarks for " + benchmark + " on fork " + parentCommand.getFork());
+      BenchmarkExecutor executor = benchmark.executorBuilder.create(output, asyncProfilerOptions);
```

Comment:
Is the path part of the options ? It would be interesting to have a path of the output file.